specification/eventgrid/data-plane/Microsoft.Media/stable/2018-01-01/MediaServices.json

```diff
@@ -366,6 +366,29 @@
       "type": "object",
       "description": "The event data for a Job output asset."
     },
+    "MediaJobOutputProgressEventData": {
+      "x-ms-discriminator-value": "#Microsoft.Media.JobOutputProgress",
```

Comment:
Didn't quite understand why the discriminator is needed for this. Can you please clarify?

Reply:
The value of x-ms-discriminator-value is how the client and server differentiate polymorphic types, which, in our case, the type of the event. You could find similar attribute in other event data like MediaJobScheduledEventData and MediaJobOutputFinishedEventData.

Reply:
Got it, thanks.